staging/src/k8s.io/component-helpers/go.mod

```diff
@@ -0,0 +1,16 @@
+// This is a generated file. Do not edit directly.
+
+module k8s.io/component-helpers
+
+go 1.15
+
+require (
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+)
+
+replace (
+	k8s.io/api => ../api
+	k8s.io/apimachinery => ../apimachinery
+	k8s.io/component-helpers => ../component-helpers
```

Comment:
this seems unnecessary? just `.` or not at all? I see it's autogenerated but haven't seen it in other places

Reply:
Yeah, it's autogenerated. Not much we can change here.